CONTRIBUTING.rst

```diff
@@ -0,0 +1,238 @@
+Contribution getting started
+============================
+
+Contributions are highly welcomed and appreciated.  Every little help counts,
+so do not hesitate! If you like tox, also share some love on Twitter or in your blog posts.
+
+.. contents:: Contribution links
+   :depth: 2
+
+.. _submitfeedback:
+
+Feature requests and feedback
+-----------------------------
+
+We'd also like to hear about your propositions and suggestions.  Feel free to
+`submit them as issues <https://github.com/tox-dev/tox/issues>`_ and:
+
+* Explain in detail how they should work.
+* Keep the scope as narrow as possible.  This will make it easier to implement.
+
+.. _reportbugs:
+
+Report bugs
+-----------
+
+Report bugs for tox in the `issue tracker <https://github.com/tox-dev/tox/issues>`_.
+
+If you are reporting a bug, please include:
```

Comment:
maybe point out here that ``tox -v`` contains most of this information, so they don't have to come up with it from the top of their head